blocks/header-nav-chain-block/chains/header-nav-chain-block/useDebounce.js

```diff
@@ -0,0 +1,84 @@
+// https://github.com/xnimorz/use-debounce
+import { useRef, useCallback, useEffect } from 'react';
+
+export default function useDebouncedCallback(callback, delay, options = {}) {
+  const { maxWait, leading, trailing = true } = options;
+  const maxWaitHandler = useRef(null);
+  const maxWaitArgs = useRef([]);
+
+  // const trailing = options.trailing === undefined ? true : options.trailing;
```

Comment:
Was this comment left here intentionally?

Reply:
is a leftover, I'll remove it